.coveragerc

```diff
@@ -1,28 +1,38 @@
 # .coveragerc to control coverage.py
 [run]
-branch = False
+source = statsmodels
+branch = True
+plugins = Cython.Coverage
 
 [report]
 show_missing = True
 
+omit=
+    */tests/*
+    */datasets/*
+
 # Regexes for lines to exclude from consideration
 exclude_lines =
     # Have to re-enable the standard pragma
     pragma: no cover
 
+    # Don't complain if tests don't hit defensive assertion code:
+    raise NotImplementedError
```

Comment:
Not necessarily for this Pr, but I think we shouldn’t gnome NotImplmentedError

Reply:
The legit reason to not count these is that Abstract classes often use NIE to force subclasses to implement methods.

Reply:
That’s reasonable, but in practice in sm it usually means someone stopped writing a function halfway through.

One option would be to use pandas AbstractMethodError for the use case you mentioned.